source/main.cpp

```diff
@@ -352,6 +351,7 @@ int main(int argc, char *argv[])
     memset(&sigset, 0, sizeof(sigset_t));
     int received_signal;
     sigaddset(&sigset, SIGINT);
+    sigaddset(&sigset, SIGTERM);
```

Comment:
Duplicate line of code. We already have `sigaddset(&sigset, SIGTERM);` on line 356

Reply:
fixed